example/geny-window.css

```diff
@@ -18,6 +18,10 @@
     margin-left: 5px;
 }
 
+.mr-1 {

```

Comment:
I think vuetify and tailwind use 4px for `mr-1`. Not sure if this is a more global standard. Is it ok to have a slightly different value?

Actually, this is just the example css, should be ok :)